src/qtgui/iq_tool.cpp

```diff
@@ -46,10 +46,13 @@ CIqTool::CIqTool(QWidget *parent) :
     is_playing = false;
     bytes_per_sample = 8;
     sample_rate = 192000;
+    playback_sample_rate = 192000;
```

Comment:
NIT: Consider rename the `sample_rate` to match the `playback_sample_rate` (eg. `record_sample_rate`). My hope is it will improve the code readability.

Reply:
Updated.